package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "wp-job-manager",
-      "version": "1.38.0",
+      "version": "1.39.0",
```

Comment:
Review note: @anaemnesis and I talked about this and are sneaking it into this PR.